From 82afbdbe4f5aafe711fe11d6a98621caadfd2487 Mon Sep 17 00:00:00 2001
From: Matt LaMantia <mlamantia@cloudera.com>
Date: Wed, 23 Mar 2016 12:30:14 -0700
Subject: [PATCH 1356/2848] YARN-4436.  DistShell
 ApplicationMaster.ExecBatScripStringtPath is
 misspelled.

Change-Id: I136dc57c076b50a00b58e9a0a39c05e326b0bc96
(cherry picked from commit 0bbaa95c0a28c5f8b44be96917e599bbfaaac653)
---
 .../distributedshell/ApplicationMaster.java        |   13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/src/main/java/org/apache/hadoop/yarn/applications/distributedshell/ApplicationMaster.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/src/main/java/org/apache/hadoop/yarn/applications/distributedshell/ApplicationMaster.java
index ec163a7..2625674 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/src/main/java/org/apache/hadoop/yarn/applications/distributedshell/ApplicationMaster.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/src/main/java/org/apache/hadoop/yarn/applications/distributedshell/ApplicationMaster.java
@@ -257,8 +257,9 @@
   private String domainId = null;
 
   // Hardcoded path to shell script in launch container's local env
-  private static final String ExecShellStringPath = Client.SCRIPT_PATH + ".sh";
-  private static final String ExecBatScripStringtPath = Client.SCRIPT_PATH
+  private static final String EXEC_SHELL_STRING_PATH = Client.SCRIPT_PATH
+      + ".sh";
+  private static final String EXEC_BAT_SCRIPT_STRING_PATH = Client.SCRIPT_PATH
       + ".bat";
 
   // Hardcoded path to custom log_properties
@@ -973,8 +974,8 @@ public void run() {
         LocalResource shellRsrc = LocalResource.newInstance(yarnUrl,
           LocalResourceType.FILE, LocalResourceVisibility.APPLICATION,
           shellScriptPathLen, shellScriptPathTimestamp);
-        localResources.put(Shell.WINDOWS ? ExecBatScripStringtPath :
-            ExecShellStringPath, shellRsrc);
+        localResources.put(Shell.WINDOWS ? EXEC_BAT_SCRIPT_STRING_PATH :
+            EXEC_SHELL_STRING_PATH, shellRsrc);
         shellCommand = Shell.WINDOWS ? windows_command : linux_bash_command;
       }
 
@@ -985,8 +986,8 @@ public void run() {
       vargs.add(shellCommand);
       // Set shell script path
       if (!scriptPath.isEmpty()) {
-        vargs.add(Shell.WINDOWS ? ExecBatScripStringtPath
-            : ExecShellStringPath);
+        vargs.add(Shell.WINDOWS ? EXEC_BAT_SCRIPT_STRING_PATH
+            : EXEC_SHELL_STRING_PATH);
       }
 
       // Set args for the shell command if any
-- 
1.7.9.5

